operator/v1/types_ingress.go

```diff
@@ -121,6 +121,22 @@ type IngressControllerSpec struct {
 	//
 	// +optional
 	NodePlacement *NodePlacement `json:"nodePlacement,omitempty"`
+
+	// bindCiphers is used to specify the cipher algorithms that are negotiated
+	// during the SSL/TLS handshake with an IngressController.
+	//
+	// If unset, the CipherProfileIntermediateType profile is used.
+	//
+	// +optional
+	BindCiphers *Ciphers `json:"bindCiphers,omitempty"`
+
+	// bindSecurityProtocol is used to specify one or more encryption protocols
+	// that are negotiated during the SSL/TLS handshake with the IngressController.
+	//
+	// If unset, all protocolTypes are used except SecurityProtocolSSLv3Version.
```

Comment:
"used"→"allowed'.  What does `protocolTypes` refer to?

Reply:
This is now `SecurityProtocolVersion`:
```
const (
	// Versions are listed in order from oldest to newest.
	//
	// SecurityProtocolSSLv3Version is v3 of the SSL security protocol
	SecurityProtocolSSLv3Version SecurityProtocolVersion = "SSLv3"
	// SecurityProtocolTLS10Version is v1.0 of the TLS security protocol.
	SecurityProtocolTLS10Version SecurityProtocolVersion = "TLSv1.0"
	// SecurityProtocolTLS11Version is v1.1 of the TLS security protocol.
	SecurityProtocolTLS11Version SecurityProtocolVersion = "TLSv1.1"
	// SecurityProtocolTLS12Version is v1.2 of the TLS security protocol.
	SecurityProtocolTLS12Version SecurityProtocolVersion = "TLSv1.2"
	// SecurityProtocolTLS13Version is v1.3 of the TLS security protocol.
	SecurityProtocolTLS13Version SecurityProtocolVersion = "TLSv1.3"
)
```